ee/clickhouse/errors.py

```diff
@@ -5,6 +5,10 @@
 from posthog.exceptions import EstimatedQueryExecutionTimeTooLong
 
 
+class CHQueryErrorCannotCompileRegexp(ServerException):
```

Comment:
We seem to be using this for multiple cases, not sure if the name reflects the right use for this. I would also suggest adding a useful default `code` & `message`

Reply:
This is basically for capturing the error CH sends to us. So, using the same name as the CH error code, to prevent confusion.

The error message we raise is `ValidationError`

Reply:
There's always a `code` and `message` generated from CH, so we shouldn't need to supply even a default one ourselves.